package.json

```diff
@@ -48,19 +48,21 @@
     "@swc/jest": "^0.2.23",
     "@types/color": "^3.0.3",
     "@types/events": "^3.0.0",
+    "@types/jest": "^29.2.5",
     "@typescript-eslint/eslint-plugin": "^5.39.0",
     "@typescript-eslint/parser": "^5.39.0",
     "eslint": "^8.25.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "jest": "^29.1.2",
+    "jest": "^29.3.1",
     "pre-commit": "^1.2.2",
     "prettier": "^2.6.1",
     "rollup": "^2.39.0",
     "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-cleaner": "^1.0.0",
     "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-resolve": "^5.2.0",
-    "typescript": "4.8.4"
+    "ts-jest": "^29.0.3",
```

Comment:
whats the reason of introducing a much slower ts-jest over swc/jest?

Reply:
No good reason - other than familiarity - i had issues with the tests not running - so just switched over to ts-jest. That said - for 2 test scripts; the overhead comparision witll be fairly trivial.

I will remove this and re-check if the testing issue still exists